gtest/src/log.rs

```diff
@@ -222,6 +222,7 @@ pub struct RunResult {
     pub(crate) others_failed: bool,
     pub(crate) message_id: MessageId,
     pub(crate) total_processed: u32,
+    pub(crate) gas_burned: Gas,
```

Comment:
btw, we can provide it similar to failed flag: `main_gas_burned` && `others_gas_burned`

Reply:
Good idea, done.